.github/workflows/sync.yml

```diff
@@ -0,0 +1,22 @@
+name: Sync upstream
+# This runs every day on 1000 UTC
+on:
+  schedule:
+    - cron: '0 10 * * *'
+  # Allows manual workflow run
+  workflow_dispatch:
+
+permissions: write-all
+
+jobs:
+ build:
+  runs-on: ubuntu-latest
+  steps:
+    - name: Create upstream version tag
+      uses: DataDog/sync-upstream-release-tag@main
+      with:
+        github_actor: "${GITHUB_ACTOR}"
+        github_repository: "${GITHUB_REPOSITORY}"
+        github_token: ${{ secrets.WORKFLOW_TOKEN }}
+        upstream_repo: debezium/debezium
+        major_version: "1"
```

Comment:
What does this `major_version` refer to? I believe Debezium is already at 2.6 so not sure if this should be `2` instead.

Reply:
Good catch! Thanks. It should definitely be "2". It's optional but I thought it would be good to use that setting to limit the blast radius in case `3.x` goes out and brings a lot of changes.

Reply:
Updated in [b0dd139](https://github.com/DataDog/debezium/pull/1/commits/b0dd1390e362e5792c71125feb4510c0cd4ecdbe)